### PR TITLE
Adding AWS_DEBUG env var to toggle amazonka debugging

### PR DESCRIPTION
**Pull request by nhibberd**
@EduardSergeev @charleso 

```
>> getDebugging
DebugDisabled
>> setEnv "MISMI_DEBUG" "true"
>> getDebugging
DebugDisabled
>> setEnv "MISMI_DEBUG" "True"
>> getDebugging
DebugEnabled
>> setEnv "MISMI_DEBUG" "False"
>> getDebugging
DebugDisabled
```

### CODE REVIEW

File: mismi-core/src/Mismi/Environment.hs [hunk 1]
```diff
@@ -53,10 +59,22 @@ getRegionFromEnv = do
     (either (throwM . RegionUnknown . T.pack) return . fromText . T.pack)
     mr
 
+getDebugging :: MonadIO m => m Debugging
+getDebugging = do
+  d <- liftIO $ lookupEnv "MISMI_DEBUG"
+  return . maybe DebugDisabled (bool DebugDisabled DebugEnabled) $ d >>= readMaybe
```
Comment (nhibberd) [inline mismi-core/src/Mismi/Environment.hs]: Unsure if we should be explicit about the `True` or is being set enough?

**Comment by markhibberd** [comment]
:100: 

Fixes https://github.com/ambiata/mismi/issues/197.

**Comment by markhibberd** [comment]
Small bikeshedding moment. Should we actually make this `AWS_DEBUG` or similar so it can be consistent across tools/libraries/languages (and be consistent with other vars).

File: mismi-core/src/Mismi/Environment.hs (hunk 1 above)
Comment (markhibberd) [inline mismi-core/src/Mismi/Environment.hs]: As discussed, probably `(== "true")` or similar instead of `readMaybe`

**Comment by charleso** [comment]
Yeah I prefer `AWS_DEBUG` too.

:100: 
:100: 
:100:

**Pull request merged**